spec/mastermind_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'mastermind'
+
+
+game = Mastermind.new
+
+RSpec.describe Mastermind, "Print Directions" do
```

Comment:
Nice use of multiple test cases...in rspec instead of having the function following the class name you can make a block called `describe '#function' do` and then have all the test cases within it. So it'd go

```
RSpec.describe Class
  describe function
     it test
```